ugit

```diff
@@ -266,6 +267,29 @@ undo_all() {
     fi
 }
 
+undo_git_tag() {
+    # Fetch all tags from remote
+    if git fetch --all --tags > /dev/null 2>&1; then
+        printf "\n Fetching tags from remote"
+    else
+        perror "Unable to fetch tags from remote.Please check repository access."
+        exit
+    fi
+    tag=$(git tag | fzf --ansi --height 80% \
+        --reverse --multi --header="Choose tag to undo" \
+        --preview "echo {} | cut -d' ' -f1 | xargs -I{} git show --color --pretty=format:%b {}" \
+        --bind 'j:down,k:up,ctrl-j:preview-down,ctrl-k:preview-up,ctrl-space:toggle-preview' --preview-window right:60% )
+
+    # Exit if no tag is selected
+    [ -z "$tag" ] && exit
+
+    # Delete git tag
+    printf "%s\nDeleting tag: $tag"
```

Comment:
Let's add a warning prompt here. Something like this

`Undoing a git tag, removes the git tag from both your local repo as well as the remote origin. Continue [N/y]: `

now if the user enters `y` or `Y`, go on with the rest of the flow